packages/url/src/test/index.js

```diff
@@ -1117,6 +1117,24 @@ describe( 'cleanForSlug', () => {
 		expect( cleanForSlug( 'the long - cat' ) ).toBe( 'the-long-cat' );
 		expect( cleanForSlug( 'the----long---cat' ) ).toBe( 'the-long-cat' );
 	} );
+
+	it( 'Should remove html entities for ampersands', () => {
+		expect(
+			cleanForSlug( 'the long cat &amp; a dog &amp;&amp; fish' )
+		).toBe( 'the-long-cat-a-dog-fish' );
+		expect( cleanForSlug( 'the long cat &amp;amp; dog' ) ).toBe(
+			'the-long-cat-amp-dog'
+		);
+	} );
+
+	it( 'Should remove html entities for less thans', () => {
```

Comment:
Likely not applicable after addressing my other feedback, but this could be a good case for [`it.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) as that could simplify the test code and remove redundancy.

Reply:
Nice! I'll add in after the discussion below, thank you!